{
	"items" : [
		{
			"url" :         "http:\/\/setob.blogspot.com\/",
			"category" :    "Repository",
			"team" :        "University_of_Washington",
			"update" :      "2008",
			"status" :      "Incomplete, unpublished, and apparently inactive",
			"contact" :     "Paramjit Singh Sandhu, Tyler Casey",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "SeToB",
			"description" : "Search engine for a few parts and devices from the Standard Registry, uploaded to a PoBoL database."
		},
		{
			"contact" :     "dbche@mail.ustc.edu.cn",
			"label" :       "Automatic Biological Circuit Design (ABCD)",
			"team" :        "USTC_Software",
			"url" :         "http:\/\/2009.igem.org\/Team:USTC_Software AND DOWNLOAD here http:\/\/sourceforge.net\/projects\/ustcabcd\/",
			"type" :        "Item",
			"year" :        "2009",
			"category" :    "biological network design",
			"description" : "Automatically design synthetic complex biological networks with desired functions defined as dynamical behavior and input-output properties. This Systems Biology Markup Language software won the Gold medal for the 2009 iGEM jamboree"
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:iHKU\/software#imagej",
			"category" :    "Image analysis",
			"team" :        "iHKU",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Image",
			"description" : "A simple Java program is used to automate the picture capturing process."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:TUDelft\/Temperature_software",
			"category" :    "RNA simulation",
			"team" :        "TUDelft",
			"dependency" :  "RNAeval, found in the Vienna RNA software package http:\/\/www.tbi.univie.ac.at\/~ivo\/RNA\/",
			"update" :      "2008",
			"status" :      "inactive",
			"contact" :     "igemtudelft @ gmail.com,",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Stability Profile Plotter",
			"description" : "The Stability Profile Plotter is a software tool that plots the free energy data that RNAeval provides as result. A template Standard Vector Graphic (svg) file, containing the grid, must be provided to the tool, which will then add the plot to this grid. Automatic production of a grid would be a nice improvement of this tool."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Chemotaxis_Modeling",
			"category" :    "Simulations of cell behavior",
			"team" :        "Tsinghua",
			"update" :      "2008",
			"status" :      "code no longer aviable",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Chemotaxis modelling",
			"description" : "Simulate the motion of a single bacterium"
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Calgary_Software\/Notebook and http:\/\/2008.igem.org\/Team:Calgary_Software\/Project",
			"category" :    "Circuit design and simulation",
			"team" :        "Calgary_Software",
			"update" :      "2008",
			"status" :      "not active \/ maybe amiable on request",
			"contact" :     "igem@ucalgary.ca",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "EvoGEM",
			"description" : "A software program that uses the registry of genetic parts provided by the iGEM competition. In this design, we used evolutionary and genetic strategies, which are useful modeling methods, especially when coupled with agent-based designs."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Edinburgh\/Software and http:\/\/www.ehmn.bioinformatics.ed.ac.uk\/igem\/",
			"category" :    "Circuit design and simulation",
			"team" :        "Edinburgh",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "MGEC (Modeling Genetically Engineered Circuits)",
			"description" : "For the inexperienced user. Through MGEC, one only needs to tell which genes and promoters are assembled in order to form their gene circuit - a preliminary model in SBML format will be automatically generated for the circuit. This model can then be analyzed in other software such as Copasi for further improvement."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Constraints_Repository, could not find download link (supposed to be here: http:\/\/research.ncl.ac.uk\/synthetic_biology\/downloads.html)",
			"category" :    "Repository",
			"team" :        "Newcastle_University",
			"update" :      "10\/29\/2008",
			"status" :      "not found",
			"contact" :     "http:\/\/2008.igem.org\/Team:Newcastle_University\/Team, lead developer: Nina Nielsen-Dzumhur",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Constraints Repository",
			"description" : "Part of a software collection designed by 2008 Newcastle iGem team. The Constraints Repository stores information about how parts work together, and associated parameters such as binding affinities, POPS, etc., where these are known. This was populated from the literature and by extracting hard-won knowledge from advisors."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Washington-Software\/Downloads",
			"category" :    "Hardware driver",
			"team" :        "Washington-Software",
			"dependency" :  "ROBOTC",
			"update" :      "October 2009 (sourceforge files not found)",
			"status" :      "not found, presumed inactive",
			"contact" :     "http:\/\/2009.igem.org\/Team:Washington-Software\/Team",
			"free" :        "Yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BioBrick-A-Bot Version 1.0",
			"description" : "Source codes for the drivers that operate the BioBrick-A-Bot built by team Washington in 2009, intended as a cheap alternative to Commercial Liquid Handling Systems, which can be really expensive. The robot was built using Lego bricks, and is designed to be modular in design, allowing for user-designed parts and added functionality. The source code needs to be read using ROBOTC firmware."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Johns_Hopkins-BAG\/Software_Tools",
			"category" :    "Sequence analysis",
			"team" :        "Johns_Hopkins-BAG",
			"update" :      "October 2009",
			"status" :      "Finished",
			"contact" :     "unknown",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "CloneQC",
			"description" : "Designed to automate the analysis of  thousands of tracefiles of sequenced DNA. The somewhat complex process  involved in calling whether or not the generated Building Block is an  exact match to the designed Building Block was automated. The program  can successfully identify exact and non-exact matches, and in some cases  where it isn\u2019t sure, it will ask a human to determine the status."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Slovenia\/Software",
			"category" :    "Nanotechnology\/Protein structure",
			"team" :        "Slovenia",
			"dependency" :  "Not found",
			"update" :      "Not found",
			"status" :      "Not found",
			"contact" :     "http:\/\/2009.igem.org\/Team:Slovenia\/Members",
			"free" :        "Not found",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Not found (nanoBricks[pro])",
			"description" : "Team Slovenia designed a method of constructing structures similar to DNA origami, but made out of polypeptides. In their page, they claim to have designed software to predict the stability of these structures--coils--, but the link to the \"software\" section reads \"...to be added soon\"."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Bologna\/Software#2",
			"category" :    "Image analysis",
			"team" :        "Bologna",
			"dependency" :  "MatLab",
			"update" :      "October 2009",
			"status" :      "Finished",
			"contact" :     "jflegias@gmail.com or andrea.samore@unibo.it",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "VIFluoR (Very Inexpensive Fluorescence Reader)",
			"description" : "Matlab code (VIFluoR) to estimate, by imaging analysis, the fluorescence emitted by a single bacterium. MatLab required"
		},
		{
			"url" :         "http:\/\/gcat.davidson.edu\/IGEM06\/oligo.html",
			"category" :    "DNA and RNA design",
			"team" :        "Davidson_Missouri_W",
			"update" :      "7\/10\/2008",
			"status" :      "active",
			"contact" :     "Lance Harden (laharden[at]davidson[dot]edu), A. Malcolm Campbell (macampbell[at]davidson[dot]edu),Laurie J. Heyer (laheyer[at]davidson[dot]edu)",
			"type" :        "Item",
			"year" :        "2007",
			"label" :       "Do-It-Yourself Gene Assembly",
			"description" : "Gene Synthesis Optimization Program"
		},
		{
			"status" :      "Finished",
			"label" :       "Moodle",
			"team" :        "Johns_Hopkins-BAG",
			"url" :         "http:\/\/baderlab.bme.jhu.edu:8888\/",
			"type" :        "Item",
			"year" :        "2009",
			"category" :    "Project management",
			"description" : "Open source course management system. It was modified to function as an online lab notebook for all B-A-G students to keep track of voluminous data and monitor progress of Building Block synthesis. Things like gel images and progress reports on the Building Blocks were stored in the MOODLE, along with various resources like protocols, lectures, and articles."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Paris\/Tool_introduction#top",
			"category" :    "Repository, LIMS",
			"team" :        "Paris",
			"dependency" :  "-",
			"update" :      "October, 2009",
			"status" :      "Unstarted, presumed inactive",
			"contact" :     "http:\/\/2009.igem.org\/Team:Paris\/Contact#top",
			"free" :        "-",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "\"iPhone Tool\"",
			"description" : "The Paris 2009 team wanted to create a Laboratory Information Management System (LIMS) (is a software that is used in the laboratory for the management of samples, laboratory users, instruments, standards and other laboratory functions such as invoicing, plate management, and workflow automation.) and database surfer iPhone app, but ran out of time."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Freiburg_software",
			"category" :    "Collaborative tool",
			"team" :        "Freiburg_software",
			"update" :      "November 2010",
			"status" :      "Finished",
			"contact" :     "Not available",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "SynBioWave",
			"description" : "Synthetic biological software suite, based on Google'ss collaboration and communication tool Wave. Made for collaborative research comprising parts design and documentation. Moreover, biologists can record and share the process of creating research data and perform basic tasks using SynBioWave. This gives synthetic biology access to the collaborative and interactive web 2.0."
		},
		{
			"url" :         "http:\/\/sourceforge.net\/projects\/keplerclotho\/",
			"category" :    "Workflow management, physical assembly",
			"team" :        "Berkeley_Software",
			"dependency" :  "Clotho",
			"update" :      "October 2009",
			"status" :      "not found, but part of the Clotho project",
			"contact" :     "http:\/\/2009.igem.org\/Team:Berkeley_Software\/Team",
			"free" :        "Yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Kepler\/Clotho integration",
			"description" : "Development of actors and automated assembly processes for the scientific workflow management system Kepler. Kepler is a multi-university design effort focusing on scientific workflows, and is used by a wide variety of projects in different fields. What the Berkely team did was to integrate Kepler into Clotho, as part of the Clotho-Eugene-Spectacles-Kepler software package"
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Bologna\/Software",
			"category" :    "DNA design",
			"team" :        "Bologna",
			"dependency" :  "MatLab",
			"update" :      "October 2009",
			"status" :      "Finished",
			"contact" :     "jflegias@gmail.com or andrea.samore@unibo.it",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BASER (Best Sequence Research by Andrea and Elisa)",
			"description" : [
				"developed to design synthetic DNA sequences whose transcribed RNAs: a) feature maximal free energy in the secondary structure (i.e. reducing the probability of its intra-molecular annealing)",
				"b) have minimal unwanted interactions with genomic mRNA",
				"c) present a minimal probability of partial\/shifted hybridization with complementary strands. These specifications are required for the proper engineering of the TRANS and CIS complementary sequences, whose functions are described in the T-REX device. MatLab required."
			]
		},
		{
			"url" :         "http:\/\/gcat.davidson.edu\/iGEM07\/genesplitter.html",
			"category" :    "DNA and RNA design",
			"team" :        "Davidson_Missouri_W",
			"update" :      "2007",
			"status" :      "active",
			"contact" :     "Karen Acker",
			"type" :        "Item",
			"year" :        "2007",
			"label" :       "Gene Splitting",
			"description" : "This page is designed to tell you what 4 primers will be used to split a gene such that hixC can be inserted within the gene'ss coding sequence"
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Berkeley_Software\/Spectacles",
			"category" :    "Visualization, sequence design",
			"team" :        "Berkeley_Software",
			"dependency" :  "Clotho, Eugene",
			"update" :      "April 2010",
			"status" :      "not found, but part of the Clotho project",
			"contact" :     "http:\/\/2009.igem.org\/Team:Berkeley_Software\/Team",
			"free" :        "Yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Spectacles",
			"description" : "A visual editing framework for the design of composite biological devices and the assignment of physical samples to functional concepts. In order to design composite devices, Spectacles provides a visual editing environment. Not only does this allow for a complete drag and drop \"schematic\" editor style design experience, but it also checks and enforces Eugene based rules, provides visual cues regarding the design'ss properties, and is completely integrated with the rest of the Clotho design environment. The latter allows the user to assign actual parts from registries to the functional concept captured by Spectacles."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:BCCS-Bristol\/Modeling-Software",
			"category" :    "Simulations of cell behavior",
			"team" :        "BCCS-Bristol",
			"update" :      "2008",
			"status" :      "active",
			"contact" :     "Mario di Bernardo and coworkers",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "BSim",
			"description" : "BSimApp is a graphical interface that enables a user to view simulations. It can be used to help users familiarise themselves with the process of defining simulations, watch small-scale simulations or verify the initial conditions of large-scale simulations. Simulations for: basic chemotaxis, adhesion, recruitment"
		},
		{
			"url" :         "http:\/\/www.cbs.dtu.dk\/services\/PHUSER\/",
			"category" :    "Primer design",
			"team" :        "DTU_Denmark",
			"update" :      "September 2011",
			"status" :      "Finished",
			"contact" :     "raz@cbs.dtu.dk",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "PHUSER (Primer Help for USER)",
			"description" : "Offers quick and easy design of PCR optimized primers for USER fusion of custom DNA fragments."
		},
		{
			"url" :         "http:\/\/2007.igem.org\/Image:Igem_code.zip and http:\/\/2007.igem.org\/Calgary\/software",
			"category" :    "Printer controle",
			"team" :        "Calgary",
			"update" :      "October 2007",
			"status" :      "active",
			"contact" :     "Pjadamia",
			"type" :        "Item",
			"year" :        "2007",
			"label" :       "PlotterPilot",
			"description" : "This program was able to conver NS-Bezier paths into a series of objective C commands that could be translated into motion by a printer system. Our system uses VHDL code to handle the FXS chip of our control ciruct."
		},
		{
			"url" :         "http:\/\/genedesign.thruhere.net\/gd\/",
			"category" :    "DNA design",
			"team" :        "Johns_Hopkins-BAG",
			"update" :      "September 2009",
			"status" :      "Finished",
			"contact" :     "notadoctor@jhmi.edu",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "GeneDesign",
			"description" : "Written by B-A-G Team members Sarah Richardson and Pablo Lee that provides tools for more specific gene or Building Block level design. Features like codon substitutions, silent site removals, restriction site design, and more are available. Most relevant to B-A-G, after the design of a gene or Building Block, tiling oligo order sheets can be automatically generated."
		},
		{
			"label" :       "SBOL-V (not software)",
			"team" :        "Stanford",
			"type" :        "Item",
			"year" :        "2009",
			"category" :    "Language definition",
			"description" : "SBOL-V is a graphical notation system that aims to complement SBOL. Synthetic Biology Open Language Visual (SBOLv) is a graphical notation that supports biological device development. It provides a formal notation for describing the physical composition of basic parts into composite parts during the development of biological devices. It is targeted for use by biological engineers in forward engineering projects. It encourages and supports model-driven engineering."
		},
		{
			"url" :         "http:\/\/www.clothocad.org\/",
			"category" :    "Integrated workflow",
			"team" :        "UC_Berkeley_Tools",
			"update" :      "Feb 2012",
			"status" :      "Feb 2008",
			"contact" :     "Douglas Densmore",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Clotho",
			"description" : "Clotho is for engineering synthetic biological systems and managing the data which is used to create them. It also provides a mechanism to begin the process of creating standardized data, algorithms, and methodologies for synthetic biology"
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Evolutionary_Algorithm, could not find download link (supposed to be here: http:\/\/research.ncl.ac.uk\/synthetic_biology\/downloads.html)",
			"category" :    "Circuit simulation\/optimization",
			"team" :        "Newcastle_University",
			"update" :      "10\/29\/2008",
			"status" :      "not found",
			"contact" :     "http:\/\/2008.igem.org\/Team:Newcastle_University\/Team, lead developer: Mark Wappett",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Evolutionary Algorithm",
			"description" : "Part of a software collection designed by 2008 Newcastle iGem team. The EA takes parts from the Parts Repository, assembles them randomly, but using constraints from the Constraints Repository, and evaluates the \"fitness\" of the resulting circuit depending on a desired input:output mapping. Equivalent parts with different parameters are swapped in and out at random, and fitter circuits selected in an iterative manner."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Bologna\/Software",
			"category" :    "Image analysis",
			"team" :        "Bologna",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Visual Fluo Bacteria",
			"description" : "a software for the analysis of bacteria fluorescence images"
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:KULeuven\/Software\/Simbiology2LaTeX",
			"category" :    "Matlab",
			"team" :        "KULeuven",
			"dependency" :  "MATLAB",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Simbiology2LaTeX (MATLAB)",
			"description" : "We'sve written a small toolbox to convert the information in the Matlab Simbiology models to LaTeX-code. This code can be easily converted to a pdf-file with various open source programs (e.g. TeXnicCenter for Windows). The idea for this toolbox is based on the plugin Squeezer for CellDesigner."
		},
		{
			"label" :       "Biostudio",
			"team" :        "Johns_Hopkins-BAG",
			"type" :        "Item",
			"year" :        "2009",
			"category" :    "DNA design",
			"description" : "Web application developed by B-A-G Team members Sarah Richardson and Pablo Lee layered on top of an existing generic genome browser called GBROWSE. BioStudio allows collaborative creation and editing of genome-sized DNA sequences, using revision control systems and most importantly, a graphical user interface, which facilitates many genome editing tasks. It provides various useful tools to aid the editor in the design, like ways to visualize the sequence at various levels of detail, or to visualize restriction site positions and recombination sites, or the percent content of certain base pairs, and so on."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Lethbridge_CCS\/Project",
			"category" :    "Protocol",
			"team" :        "Lethbridge_CCS",
			"update" :      "30 October 2008",
			"status" :      "Finished",
			"contact" :     "http:\/\/2008.igem.org\/Team:Lethbridge_CCS\/Team, contact data not found",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Not a software, but a proposed protocol.",
			"description" : "Now that there is a standard definition of  BioBrick, this is a ligase-independent cloning protocol (based on the technique of Aslanidis & de Jong (1990)), proposed as a standard method for BioBrick preparation."
		},
		{
			"url" :         "http:\/\/synbioss.org",
			"category" :    "Integrated workflows",
			"team" :        "Minnesota",
			"update" :      "Apr 2011",
			"status" :      "recent",
			"contact" :     "Yiannis Kaznessis",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "SynBioSS",
			"description" : "The Synthetic Biology Software Suite (SynBioSS) is a software suite for the generation, storing, retrieval and quantitative simulation of synthetic biological networks. SynBioSS facilitates computational synthetic biology and consists of three independent components: the Desktop Simulator (DS), the Wiki, and the Designer."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Workbench, could not find download link (supposed to be here: http:\/\/research.ncl.ac.uk\/synthetic_biology\/downloads.html)",
			"category" :    "Circuit design",
			"team" :        "Newcastle_University",
			"update" :      "10\/29\/2008",
			"status" :      "not found",
			"contact" :     "http:\/\/2008.igem.org\/Team:Newcastle_University\/Team, Workbench lead developer: Morgan Taschuk",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Workbench",
			"description" : "Part of a software collection designed by 2008 Newcastle iGem team. The Workbench Module is a graphical front end that allows circuits to be designed by hand, using drag-and-drop of icons, and drawing upon the Parts Repository and the Constraints Repository. The designed circuits can then be sent to the Evolutionary Algorithm to be further refined."
		},
		{
			"url" :         "http:\/\/2007.igem.org\/Imperial\/Dry_Lab\/Software",
			"category" :    "various",
			"team" :        "Imperial",
			"update" :      "10\/27\/2007",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2007",
			"label" :       "several routines in MATLAB",
			"description" : "several routines in MATLAB"
		},
		{
			"url" :         "genomicon.ca (software not found)",
			"category" :    "Educational",
			"team" :        "Alberta",
			"dependency" :  "No",
			"update" :      "Not found",
			"status" :      "Active",
			"contact" :     "http:\/\/2010.igem.org\/Team:Alberta\/team",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "GENOMIKON",
			"description" : "Building DNA is too hard. It has to be cheaper, faster, and easier, so that everyone can do it. So... we'sve created an inexpensive, self-contained educational kit called GENOMIKON - everything you need to build, test and operate genetic circuits and metabolic pathways. It comes with an online lab manual, notebook, and experiment software. It can be used by professionals as well."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Illinois-Tools\/Project",
			"category" :    "Metabolic engineering",
			"team" :        "Illinois-Tools",
			"update" :      "June 2010",
			"contact" :     "Palak Doshi,     Donny Amfahr",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "Interactive Metabolic Pathway Tools (IMP Tools)",
			"description" : "Open source, web based program that involves model-guided cellular engineering where new metabolic functions can be added to existing microorganisms. This program will assist in the design stage of synthetic biology research. IMP tools is written primarily in python using the Django web framework. It takes a user-defined input compound, output compound, and weighting scheme and determines the ideal pathway from the starting to the ending compound."
		},
		{
			"status" :      "Actve",
			"label" :       "DIY-Gem",
			"team" :        "Baltimore_US",
			"type" :        "Item",
			"year" :        "2010",
			"category" :    "low-cost high-throughput gene synthesis",
			"description" : "Our project expands the accessibility of Taq polymerase by engineering it to BioBrick standards. This allows for the expression and recovery of polymerase from transformed E. coli at a fraction of the cost of highly purified commercial enzyme."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:KULeuven\/Software\/MultiCell",
			"category" :    "Matlab",
			"team" :        "KULeuven",
			"dependency" :  "MATLAB",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Multi-cell toolbox (Matlab)",
			"description" : "Have you ever wanted to simulate the effects of your MATLAB Symbiology project in a multicellular environment This toolbox enables you to expand your 'swhat happens in 1 cell's-model into a network of interconnected cells with the full abilities of a growing population of cells. It delivers you an easy-to-use graphical interface to configure the simulation settings and with just 1 click an extensive simulation is started."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Parts_Repository, could not find download link (supposed to be here: http:\/\/research.ncl.ac.uk\/synthetic_biology\/downloads.html)",
			"category" :    "Repository",
			"team" :        "Newcastle_University",
			"update" :      "10\/29\/2008",
			"status" :      "not found",
			"contact" :     "http:\/\/2008.igem.org\/Team:Newcastle_University\/Team, lead developer, Megan Aylward",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "Parts Repository",
			"description" : "Part of a software collection designed by 2008 Newcastle iGem team. The Parts Repository Modulestores information from the literature and various databases about biological parts such as promoters, ribosomal binding sites and protein coding regions. As well as biological information, this includes fragments of CellML that facilitate incorporation of the part into a larger simulation."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:UNIPV-Pavia\/Software AND DOWNLOAD here http:\/\/aimed11.unipv.it\/SequenceAlignment\/",
			"category" :    "sequence alignment",
			"team" :        "UNIPV-Pavia",
			"update" :      "October 2009",
			"status" :      "Finished",
			"contact" :     "paolo.magni@unipv.it",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "SequenceAlignment",
			"description" : "Align sequencing results with expected sequences. Assembling simple parts.Matlab required"
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Calgary\/Modelling\/MC\/Intro, download not found",
			"category" :    "Bacteria simulation",
			"team" :        "Calgary",
			"dependency" :  "not found",
			"update" :      "not found",
			"status" :      "not found",
			"contact" :     "http:\/\/2009.igem.org\/Team:Calgary\/Team",
			"free" :        "not found",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "MCMGem",
			"description" : "This model allows us to observe the behavior of each individual cell as well as the emergent properties of the whole population. It also enables us to manipulate factors involved in the simulation to understand their effects in individual as well as colony behaviors. Based on the Membrane Computing technique"
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:Bologna\/Software",
			"category" :    "Repository",
			"team" :        "Bologna",
			"update" :      "2008",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "ARQ",
			"description" : "a Java tool to query the Registry"
		},
		{
			"label" :       "SynBioSS Designer",
			"team" :        "Minnesota",
			"type" :        "Item",
			"year" :        "2009",
			"description" : "See row 53. The 2009 Minnesota team'ss project was to overhaul the SynBioSS UI and add new functionality."
		},
		{
			"url" :         "http:\/\/2008.igem.org\/Team:TUDelft\/Temperature_software",
			"category" :    "RNA design",
			"team" :        "TUDelft",
			"update" :      "2008",
			"status" :      "inactive",
			"contact" :     "igemtudelft @ gmail.com,",
			"type" :        "Item",
			"year" :        "2008",
			"label" :       "RNA Hairpin Designer",
			"description" : "The RNA hairpin designer is a software tool that designs RNA sequences that will form a hairpin structure with a given stability profile. This way, a temperature sensitive hairpin can be designed automatically."
		},
		{
			"url" :         "http:\/\/eugenecad.org\/",
			"category" :    "Language definition",
			"team" :        "Berkeley_Software",
			"dependency" :  "Best with Clotho",
			"update" :      "March 2010",
			"status" :      "active, possibly under new name",
			"contact" :     "Douglas Densmore",
			"free" :        "Yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Eugene",
			"description" : "Eugene - Biological programming language to constrain the composition of genetic parts as a specification of biological constructs. One can both specify the structure of the design (its basic elements) as well composites of basic elements. A system of rules for part composition is also provided."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Valencia\/Simulations",
			"category" :    "Cell Simulation",
			"team" :        "Valencia",
			"dependency" :  "MATLAB",
			"update" :      "August, 2009",
			"status" :      "Finished",
			"contact" :     "http:\/\/2009.igem.org\/Team:Valencia\/TeamVal",
			"free" :        "Yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "\"Modelling Interface\"",
			"description" : "A MATLAB-based simulation that predicts how intracellular calcium concentration changes through time in cardiomyocites and neurons, according to a set of given parameters."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:CBNU-Korea\/Project",
			"category" :    "DNA design",
			"team" :        "CBNU-Korea",
			"update" :      "October 2009",
			"status" :      "Not available",
			"contact" :     "koreashadow@gmail.com, x2zerg@hanmail.net, gkgkeovh@naver.com",
			"type" :        "Item",
			"year" :        "2009",
			"label" :       "Essarker: An Essential Remarker for a Minimal for Synthetic Genome",
			"description" : "The goal of Essarker is to help users design a minimal genome synthesized through the fundamental frame comprising the essential genes of replication. Essarker is a standalone software to manage and retrieve required sequences of genomes, and explore the essential gene order and direction and the related orthologous genes. It also identifies and visualizes the positions and orientations of genes. In addition, it shows optimal ordering of essential genes and orthologs by statistical analysis."
		}
	]
}
