{
	"items" : [
		{
			"url" :         "http:\/\/2011.igem.org\/Team:Sevilla\/Software\/GPSE",
			"category" :    "Circuit Design, Simulation",
			"team" :        "Sevilla",
			"update" :      "2011",
			"status" :      "under development",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "GPSE (Graphical P-System Editor)",
			"description" : "CAD to design biological P-Systems. The main goal of our program is to make the preparation of the files needed for the P-Systems simulators easier, more intuitive"
		},
		{
			"url" :         "http:\/\/120.126.44.58\/SynthoPrime\/ and http:\/\/2011.igem.org\/Team:NYMU-Taipei\/synthoprime",
			"category" :    "Repository",
			"team" :        "NYMU-Taipei",
			"update" :      "2011",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "SynthoPrime",
			"description" : "A Synthetic Biology Primer Design Tool for Genomic Parts Creation"
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Illinois-Tools\/Project",
			"category" :    "Metabolic Engineering",
			"team" :        "Illinois",
			"dependency" :  "Python, Django, MySQL",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Nathan Price, Kaustubh Bhalerao",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "IMP Tools - Interactive Metabolic Pathway Tools",
			"description" : "Determines the ideal pathway from the starting to the ending compound. For model-guided cellular engineering where new metabolic functions can be added to existing microorganisms. Assists in the design stage of synthetic biology research. Takes a user-defined input compound, output compound, and weighting scheme."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Constraints_Repository",
			"category" :    "Repository",
			"team" :        "Newcastle University",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Anil Wipat",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Constraints Repository",
			"description" : "Stores information about how parts work together, and associated parameters such as binding affinities, POPS, etc., where these are known. This was populated from the literature and by extracting hard-won knowledge from advisors."
		},
		{
			"source" :      "http:\/\/2010.igem.org\/Team:Imperial_College_London\/Software_Tool",
			"url" :         "http:\/\/2010.igem.org\/Team:Imperial_College_London\/Software_Tool",
			"category" :    "Reference",
			"team" :        "Imperial College London",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Guy-Bart Stan",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Parasight Software Tool",
			"description" : "Web display of the custom sequence of the entire surface protein construct with sensitivity to different proteases. By changing the cleavage site the system can accept a wide variety of inputs. Presents the user with a sequence designed to create a \"detection model\" sensitive to one of ten pathogen proteases."
		},
		{
			"source" :      "http:\/\/cloneqc.thruhere.net\/, http:\/\/128.220.136.46\/wiki\/baderlab\/index.php\/Servers",
			"url" :         "http:\/\/2009.igem.org\/Team:Johns_Hopkins-BAG\/Software_Tools",
			"category" :    "Sequence Analysis",
			"team" :        "Johns Hopkins",
			"dependency" :  "Web",
			"update" :      "2009",
			"status" :      "active",
			"contact" :     "Sarah Richardson, Jef Boeke, Joel Bader",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "CloneQC",
			"description" : "Automates the analysis of thousands of tracefiles of sequenced DNA. Automatically calls whether or not the generated Building Block is an exact match to the designed Building Block. The program can successfully identify exact and non-exact matches, and in some cases where it isn\u2019t sure, it will ask the operator of the software to determine the status."
		},
		{
			"source" :      "https:\/\/github.com\/haydnKing\/Gibthon",
			"url" :         "http:\/\/gibthon.org\/",
			"category" :    "Assembly Planning, Gibson Assembly",
			"team" :        "Cambridge",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Haydn King, Bill Collins",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gibthon Ligation Calculator",
			"description" : [
				"A simple tool for calculating reactant concentrations for DNA ligation for Gibson Assembly",
				"Ligation ratio calculator for Gibson Assembly. Set the volumes and add components to calculate final volumes of the Ligation Mix."
			]
		},
		{
			"source" :      "http:\/\/code.google.com\/p\/crispr-studio\/",
			"url" :         "http:\/\/2011.igem.org\/Team:Arizona_State\/Project\/Software",
			"category" :    "Sequence Generation",
			"team" :        "Arizona State",
			"dependency" :  "BLAST+",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Xiao Wang",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "CRISPRstudio",
			"description" : "\"We have developed a tool to assist in the development of synthetic CRISPR systems. Pick spacers from a source sequence, based on homology with the target genome, hairpinning potential, restriction sites, and known PAMs. Output arrays with various formats based on generated spacers. Display and gather CRISPR information using a database cultivated from several sources.CRISPRstudio is provided \"as is\" with no express or implied warranty for accuracy or accessibility.\""
		},
		{
			"source" :      "https:\/\/github.com\/jkdirac\/igame",
			"url" :         "http:\/\/2010.igem.org\/Team:USTC_Software\/Features",
			"category" :    "Game, Educational, Simulation, Modelling",
			"team" :        "USTC",
			"dependency" :  "COPASI",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Hao Jiang",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "iGaME",
			"description" : "A game to instruct non-biologists to design and improve biological systems. Players help solve complex designs of biological systems by submitting the assembly of parts and the software discovers and generates the biological model automatically. Leverages the MoDeL language (for more see BBF RFC55: Standard Biological Part Automatic Modeling Database Language (MoDeL)) to generate the model for simulation using COPASI."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Berkeley_Software\/Spectacles",
			"category" :    "Visualization, Sequence Design",
			"team" :        "Berkeley",
			"dependency" :  "Clotho, Eugene",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Spectacles",
			"description" : "A visual editing framework for the design of composite biological devices and the assignment of physical samples to functional concepts. In order to design composite devices, Spectacles provides a visual editing environment. Not only does this allow for a complete drag and drop \"schematic\" editor style design experience, but it also checks and enforces Eugene based rules, provides visual cues regarding the design's properties, and is completely integrated with the rest of the Clotho design environment. The latter allows the user to assign actual parts from registries to the functional concept captured by Spectacles."
		},
		{
			"source" :      "http:\/\/2008sw.igem.org\/bccs_bristol\/src\/BSim_Source.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:BCCS-Bristol\/Modeling-Software",
			"category" :    "Simulation, Visualization",
			"team" :        "BCCS-Bristol",
			"dependency" :  "Java",
			"update" :      "2008",
			"status" :      "active",
			"contact" :     "Mario di Bernardo",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "BSim",
			"description" : "View simulations in a graphical interface. It can be used to help familiarize yourself with the process of defining simulations, watch small-scale simulations, or verify the initial conditions of large-scale simulations. Simulations for: basic chemotaxis, adhesion, recruitment"
		},
		{
			"source" :      "http:\/\/2007.igem.org\/Image:Constitutive_Promoter_Analysis.zip",
			"url" :         "http:\/\/2007.igem.org\/Imperial\/Dry_Lab\/Software",
			"category" :    "Data Analysis",
			"team" :        "Imperial",
			"dependency" :  "MATLAB",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Richard Kitney, Paul Freemont",
			"free" :        "yes",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "Constitutive Synthesis of a Protein",
			"description" : [
				"The experimental setup involves tagging a constitutive promoter with a fluorescent protein",
				"the resultant fluorescence is then recorded and it is up to the user now to perform some analysis. This routine loads the experimental data as an excel sheet, allows for graphic visualization of the results, and if need be, elimination of some samples which may seem suspect (outliers). The synthesis rate and degradation term of the milieu are defined\/estimated by the user. The routine was developed to impart control to the user over all the operations."
			]
		},
		{
			"source" :      "http:\/\/genedesign.thruhere.net\/gd\/",
			"url" :         "http:\/\/genedesign.thruhere.net\/gd\/",
			"category" :    "DNA Design",
			"team" :        "Johns Hopkins",
			"dependency" :  "Web",
			"update" :      "2009",
			"status" :      "active",
			"contact" :     "Sarah Richardson, Jef Boeke, Joel Bader",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "GeneDesign",
			"description" : "Aid gene or Building Block level design. For example, codon substitutions, silent site removals, restriction site design, etc. The design of a gene or Building Block, tiling oligo order sheets can be automatically generated."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:CBNU-Korea\/Project",
			"category" :    "DNA Design",
			"team" :        "CBNU-Korea",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Young-Chang Kim, Chung Sei Rhee, Sung Duck Lee, Sunshin Kim",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Essarker - An Essential Remarker for a Minimal for Synthetic Genome",
			"description" : [
				"Design a minimal genome synthesized through the fundamental frame comprising the essential genes of replication. A standalone software tool to manage and retrieve required sequences of genomes",
				"explore the essential gene order and direction",
				"and the related orthologous genes. It also identifies and visualizes the positions and orientations of genes. In addition, it shows optimal ordering of essential genes and orthologs by statistical analysis."
			]
		},
		{
			"source" :      "http:\/\/biostudio.bme.jhu.edu:8080\/",
			"url" :         "http:\/\/biostudio.bme.jhu.edu\/home\/",
			"category" :    "DNA Design",
			"team" :        "Johns Hopkins",
			"dependency" :  "GBrowse, Apache, BioPerl, MySQL",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Sarah Richardson, Jef Boeke, Joel Bader",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BioStudio",
			"description" : "Facilitates many genome creation and editing tasks in collaborative environment on the web. Aids the editor in the design, visualization of the sequence at various levels of detail. For example to view restriction site positions and recombination sites, and the percent content of certain base pairs. Uses a revision control system. Layered on top of an existing generic genome browser called GBROWSE."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/clothocad\/",
			"url" :         "http:\/\/www.clothocad.org\/",
			"category" :    "Integrated Workflow",
			"team" :        "UC Berkeley",
			"dependency" :  "Netbeans",
			"update" :      "2012",
			"status" :      "active",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Clotho",
			"description" : "Engineer synthetic biological systems and manage the data which is used to create them. A mechanism to begin the process of creating standardized data, algorithms, and methodologies for synthetic biology."
		},
		{
			"source" :      "http:\/\/2007.igem.org\/Image:Degradation_Term_Analysis.zip",
			"url" :         "http:\/\/2007.igem.org\/Imperial\/Dry_Lab\/Software",
			"category" :    "Data Analysis",
			"team" :        "Imperial",
			"dependency" :  "MATLAB",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Richard Kitney, Paul Freemont",
			"free" :        "yes",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "Analysis of the Degradation of a Protein",
			"description" : "The experimental setup involves obsering the natural degradation of a fluorescent protein with a fluorometer. . This routine loads the experimental data as an excel sheet, allows for graphic visualization of the results, and if need be, elimination of some samples which may seem suspect (outliers). The degradation term of the milieu is then estimated by shape matching. The routine was developed to impart control to the user over all the operations."
		},
		{
			"source" :      "http:\/\/github.com\/douglas-watson\/muigi",
			"url" :         "http:\/\/2011.igem.org\/Team:EPF-Lausanne\/Tools\/Microfluidics\/Tamagotchip",
			"category" :    "Game, Microfluidics",
			"team" :        "EPF-Lausanne",
			"dependency" :  "Microfludics hardware",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Douglas Watson",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Tamagotchip Game\/ Muigi the Microplumber",
			"description" : "The tamagotchip live online microfluidics game,  a web-controlled microfluidics setup, where users can view the chip live and control the valves from a web browser. From a web browser, control the setup located in our lab in Lausanne."
		},
		{
			"source" :      "http:\/\/igem.molgenrug.nl\/igem2011\/Cumulus\/Cumulus.zip",
			"url" :         "http:\/\/2011.igem.org\/Team:Groningen\/modeling_cumulus",
			"category" :    "Modelling, Data Sharing",
			"team" :        "Groningen",
			"dependency" :  "no",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Matthias Heinemann",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Cumulus",
			"description" : "Cloud based modelling and data exchange program that we developed. Its purpose is to show, as a proof of concept, that it is possible to integrate both the exchange of data, the production of comprehensive models, and the sharing of computational resources into one platform. collaborative Biology Framework based on Service Oriented Architectures (SOA) and on a Web Services. This approach simplifies and harmonizes the work carried out on the Cloud and Grid platform by geographically dispersed scientists. The new collaborative SOA Cloud Framework (called Cumulus) provides an efficient and easy way to run distributed software.This capability is put to use in a parameter optimization program, (currently a genetic algorithm) to find the optimal parameters for submitted models. In other words, Cumulus makes it simple to use cloud computing resources to find parameters for models, and at the same time, leads to the sharing of information."
		},
		{
			"source" :      "https:\/\/github.com\/stjahns\/Alberta-IGEM",
			"url" :         "http:\/\/www.genomikon.ca\/",
			"category" :    "Educational, E-labnotebook, Lab manual",
			"team" :        "Alberta",
			"dependency" :  "Web",
			"update" :      "Not found",
			"status" :      "finished",
			"contact" :     "Mike Ellison, Doug Ridgway",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "GENOMIKON",
			"description" : "An interactive lab manual which walks a student through the laboratory experiment. The lab notebook component empowers students to create their own experiments and share them online. It can be used by professionals as well. The web app compliments the GENOMIKON educational kit- everything needed to build, test and operate genetic circuits and metabolic pathways."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/ustcabcd\/",
			"url" :         "http:\/\/2009.igem.org\/Team:USTC_Software",
			"category" :    "Network Design",
			"team" :        "USTC",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Bo Ding",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "ABCD - Automatic Biological Circuit Design",
			"description" : "Automatically design synthetic complex biological networks with desired functions defined as dynamical behavior and input-output properties. Uses SBML."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Bologna\/Software",
			"category" :    "DNA Design",
			"team" :        "Bologna",
			"dependency" :  "MATLAB",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Emanuele Giordano, Giulio Zanaroli",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BASER - Best Sequence Research by Andrea and Elisa",
			"description" : [
				"Design synthetic DNA sequences whose transcribed RNAs: a) feature maximal free energy in the secondary structure (i.e. reducing the probability of its intra-molecular annealing)",
				"b) have minimal unwanted interactions with genomic mRNA",
				"c) present a minimal probability of partial\/shifted hybridization with complementary strands. These specifications are required for the proper engineering of the TRANS and CIS complementary sequences, whose functions are described in the T-REX device."
			]
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Paris\/Tool_introduction#top",
			"category" :    "Repository, LIMS",
			"team" :        "Paris",
			"dependency" :  "iOS",
			"update" :      "2009",
			"status" :      "not found",
			"contact" :     "Fran\u00E7ois Le F\u00E8vre",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "iPhone Tool",
			"description" : "Management of samples, laboratory users, instruments, standards and other laboratory functions such as invoicing, plate management, and workflow automation.) and database surfer iPhone app"
		},
		{
			"url" :         "http:\/\/www.clothocad.org\/",
			"category" :    "Integrated Workflow",
			"team" :        "BU-Wellesley",
			"dependency" :  "no",
			"update" :      "2012",
			"status" :      "active",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Clotho 2.0 updated",
			"description" : "Clotho is a software platform for synthetic biologists. In this update created a software suite for performing automated software assembly. We also made a primer designing tool,Genbank parsing tool, a tool for viewing sequence annotations. And we redesigned the interface to make Clotho easier to use and more effective as a tool for researchers. All of these tools can interact seamlessly with the same data. Clotho is the converging point for the tools we have made at Boston University, as well as those being developed at Johns Hopkins University, UC Berkeley, and tools we have not even dreamed up yet. Clotho is a nexus. Clotho is the foundation for the automated future of biological engineering. All of these feats are accomplished without Clotho doing anything itself. At its core, Clotho is a data model and a plugin infrastructure. In other words, Clotho gives a vocabulary to describe biological objects and a way for many different types of tools to interact with these biological objects. By itself, Clotho is not a tool. However, the plugin infrastructure allows Clotho to be a rallying points for all sorts of different tools, each fulfilling the exact need of the user."
		},
		{
			"source" :      "http:\/\/synbiowave.sourceforge.net, http:\/\/www.synbiowave.org\/",
			"url" :         "http:\/\/2010.igem.org\/Team:Freiburg_Software",
			"category" :    "Collaboration, Information Sharing, Integrated Framework",
			"team" :        "Freiburg",
			"dependency" :  "Google Wave",
			"update" :      "Google Wave closed",
			"status" :      "finished",
			"contact" :     "Kristian M\u00FCller",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "SynBioWave 2.0",
			"description" : "Research collaboration by real-time sharing of parts, design and documentation. Moreover, biologists can record and share the process of creating research data. Improved user friendliness, separated the input and output from the sequence database operations by creating a linked wave for data storage. New add-ons that perform specific external tasks and feed back into the main program: 1. The \u201Cblueprint-robot\u201D, a framework easing new robot development. 2. Adding new functionality by creating add-on robots that perform tasks such as BLAST-searches, ORF-finding, translation, sequence alignments and restriction site mapping."
		},
		{
			"source" :      "http:\/\/2011.igem.org\/Team:HUST-China\/Download",
			"url" :         "http:\/\/2011.igem.org\/Team:HUST-China",
			"category" :    "Modelling, Circuit Design, Educational",
			"team" :        "HUST-China",
			"dependency" :  "Flash, Papervision 3D, Php, My SQL",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Ge,Qian",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "SEA - Super E.coli Architect",
			"description" : [
				"Frees biological designers from technical details by helping them work as architects. Get to know synthetic biology",
				"Create functional element with new function",
				"Design bacteria",
				"Generate gene circuits",
				"Plan the experiment",
				"Export systematic biology markup language(SBML). Metabolic Pathway Search will compute the optimal pathway for the best response parameter, including the enzyme, etc."
			]
		},
		{
			"source" :      "http:\/\/2010.igem.org\/Team:Cambridge\/Tools\/Eglometer",
			"url" :         "http:\/\/2010.igem.org\/Team:Cambridge\/Tools\/Eglometer",
			"category" :    "Hardware Driver, Operation, Construction Instructions",
			"team" :        "Cambridge",
			"dependency" :  "Hardware, Arduino libraries: Keypad.h LiquidCrystal.h",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jim Ajioka,Jim Haseloff, Gos Micklem",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "E.glometer",
			"description" : "A low cost electronic system for measuring light output, this is useful for reporter assays. Instructions on how to build and use the E.glometer are provided, along with the Arduino source code."
		},
		{
			"source" :      "http:\/\/clothocad.svn.sourceforge.net\/",
			"url" :         "http:\/\/www.clothocad.org\/",
			"category" :    "Integrated workflows",
			"team" :        "Berkeley",
			"update" :      "2012",
			"status" :      "active",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Clotho 2.0",
			"description" : "Added the \"apps\" functionality to Clotho"
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/d\/d0\/StabilityProfilePlotter.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:TUDelft\/Temperature_software",
			"category" :    "RNA Simulation",
			"team" :        "TU Delft",
			"dependency" :  "RNAeval, Vienna RNA",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Alexander Gultyaev, Filippo Menolascina",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Stability Profile Plotter",
			"description" : "Plots the free energy data that RNAeval provides as result. A template Standard Vector Graphic (svg) file, containing the grid, must be provided to the tool, which will then add the plot to this grid. Automatic production of a grid would be a nice improvement of this tool."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Chemotaxis_Modeling",
			"category" :    "Simulation",
			"team" :        "Tsinghua",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Xue Gao, Guoqiang Chen, Zhirong Sun",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Chemotaxis modelling",
			"description" : "Simulate the motion of a single bacterium"
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2011.igem.org\/Team:Dundee\/Software",
			"category" :    "DNA Sequence Design",
			"team" :        "Dundee",
			"dependency" :  "Android, iOS",
			"update" :      "2011",
			"status" :      "finished",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "The Gene Slicer",
			"description" : "Helps restriction mapping of nucleotide sequences and to search a sequence for a specific enzyme and can also return a list of enzymes not present in the sequence."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2007.igem.org\/Calgary\/evoGEM_introduction",
			"category" :    "Circuit Design, Evolutionary Algorithm",
			"team" :        "Calgary",
			"dependency" :  "Vigo::3D",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Christian Jacob",
			"free" :        "not available",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "EvoGEM",
			"description" : "Find genetic circuits that would produce a desired behavior before these are tested in a lab. Uses evolutionary design,effecient mutation operators on these circuits and an effecient fitness function that can solve the relatively simple problem of synthesizing a requested molecule considering the existing iGEM bio-bricks."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:BU_Wellesley_Software\/eLabNotebook",
			"category" :    "Laboratory, LIMS",
			"team" :        "BU-Wellesley",
			"dependency" :  "iOS",
			"update" :      "2011",
			"status" :      "design",
			"contact" :     "Douglas Densmore",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "eLabNotebook",
			"description" : "An electronic lab notebook that facilitates collaborative, effective, and safe assembly of biological systems. The eLabNotebook allows users to: 1. Capture multimodal data inside and outside the lab with minimal effort 2. Follow a protocol in the lab while modifying and annotating steps as needed 3. Access a wide variety of digital resources 4. Get an overview of current activity and progress 5.share physical and digital resources in real-time"
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:Sevilla\/Software\/Ubbit",
			"category" :    "Database",
			"team" :        "Sevilla",
			"update" :      "2011",
			"status" :      "under development",
			"contact" :     "Eduardo Villalobo, Francisco Romero, Ana Calvo",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "Ubbit repository",
			"description" : "Web database developed to facilitate the implementation of the Ubbit standard."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Alberta\/Project\/Automation",
			"category" :    "Robotics",
			"team" :        "Alberta",
			"dependency" :  "nxtOSEK, GNUARM",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Chris Backhouse, Robert Campbell, Linda Reha-Krantz, Tracy Raivio, Jon Dennis",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BioBytes Assembly System",
			"description" : "Drive a custom-built robot for assembly"
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:NYC_Software\/Tools\/Primer_Designer and https:\/\/github.com\/igemsoftware\/NYC_Software_2011",
			"category" :    "Repository",
			"team" :        "NYC",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Russell Durrett",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "Primer Designer",
			"description" : "designing primers for over 50 future biobricks. In order to aid them, and of course future iGEMers, we developed a command line tool for creating primer sequences suitable for biobricking under RFC23 - the Silver Standard"
		},
		{
			"source" :      "http:\/\/ethzigem10.svn.sourceforge.net\/viewvc\/ethzigem10\/",
			"url" :         "http:\/\/2010.igem.org\/Team:ETHZ_Basel\/Achievements\/Matlab_Toolbox",
			"category" :    "MATLAB Algorithms, Modelling",
			"team" :        "ETHZ Basel",
			"dependency" :  "Matlab, Simulink",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Sven Panke, J\u00F6rg Stelling",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Lemming Toolbox",
			"description" : "Bundle of Matlab algorithms useful for biology. Complex models, as well as novel visualization, user input and microscope control approaches, as well as a modular graphical user interface based on Simulink, to aid in design of models of any nature."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/9\/94\/VFB1.0.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:Bologna\/Software",
			"category" :    "Image Analysis",
			"team" :        "Bologna",
			"dependency" :  "MATLAB",
			"update" :      "2008",
			"status" :      "active",
			"contact" :     "Marco Caprini, Emanuele Giordano",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Visual Fluo Bacteria",
			"description" : "Analysis of bacteria fluorescence images"
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2011.igem.org\/Team:Dundee\/Software",
			"category" :    "DNA Sequence Design",
			"team" :        "Dundee",
			"dependency" :  "Android, iOS",
			"update" :      "2011",
			"status" :      "finished",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "The Gene Cutter",
			"description" : "Helps restriction mapping of nucleotide sequences and to search a sequence for a specific enzyme and can also return a list of enzymes not present in the sequence."
		},
		{
			"source" :      "http:\/\/2010.igem.org\/wiki\/images\/6\/61\/VTENSIMAGBiosecurity_PCR_src.zip",
			"url" :         "http:\/\/2010.igem.org\/Team:VT-ENSIMAG\/User_primer",
			"category" :    "Primer Design",
			"team" :        "VT-ENSIMAG",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jean Peccoud",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "User Primer Calculator",
			"description" : "Finds 4 primers: the forward primer overlapping with the first input, the reverse user primer that overlaps with the first sequence and the gap sequence added on the PCR, the forward user sequence, overlapping with the second sequence and the gap sequence, and endly the reverse primer, which overlaps with the second sequence. If some suffix, prefix or terminator are added on the full sequence, they will be added in the forward and reverse primers. To detect the length of the overlapping part of the primers, we search for the correct length that will have the desired melting temperature. This melting temperature is calculated. The find the position of the cutting enzyme (where the nucleotide U was placed), we search for an A in the gap sequence, and a matching T 8~13bp further. The A will be replaced by the U in the reverse user primer, and the T will be replace by the U in the forward user primer."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Workbench",
			"category" :    "Circuit Design",
			"team" :        "Newcastle University",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Anil Wipat",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Parts Repository Module",
			"description" : "Circuit design by hand, a graphical interface using drag-and-drop of icons, and drawing upon the Parts Repository and the Constraints Repository. The designed circuits can then be sent to the Evolutionary Algorithm to be further refined."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/django.gibthon.org\/tools\/ligcalc\/",
			"category" :    "Calculator",
			"team" :        "Cambridge",
			"dependency" :  "PHP, web, Django, mfold, BioPython, jQuery",
			"update" :      "2012",
			"status" :      "finished",
			"contact" :     "Jim Ajioka,Jim Haseloff, Gos Micklem",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Ligation Calculator",
			"description" : "Helps calculate the volumes of DNA used for ligation."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/ustc-igame\/files\/",
			"url" :         "http:\/\/2010.igem.org\/Team:USTC_Software\/MoDeL",
			"category" :    "Modelling, Database",
			"team" :        "USTC",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Hao Jiang",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "MoDeL -update",
			"description" : "Rule-based modeling language and a database of templates to create models of gene circuits or reaction networks. Parts, biochemical reactions and species, \"etc\" and combined to form an SBML file. Templates are used to enforce functional \"rules\". Note: The documentation is lacking in clarity and does not explain the relationship of this approach to other well known modelling and simulation research."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Team:Calgary_Software\/Project",
			"category" :    "Circuit Design, Simulation",
			"team" :        "Calgary",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Christian Jacob, Anders Nygren",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "EvoGEM GUI",
			"description" : "Uses the registry of genetic parts provided by the iGEM competition. Evolutionary and genetic strategies, which are useful modeling methods, especially when coupled with agent-based designs.Models the processes within the cell, optimizes BioBrick circuits.  a more sophisticated system, both in terms of data retrieval and modeling. Moreover, we developed a user interface that makes the software easy-to-use"
		},
		{
			"source" :      "http:\/\/aimed11.unipv.it\/SequenceAlignment\/",
			"url" :         "http:\/\/2009.igem.org\/Team:UNIPV-Pavia\/Software",
			"category" :    "Sequence Alignment",
			"team" :        "UNIPV-Pavia",
			"dependency" :  "MATLAB",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Paolo Magni",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "SequenceAlignment",
			"description" : "Align sequencing results with expected sequences. Assembling simple parts."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/e\/e0\/ARQ1.1source.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:Bologna\/Software",
			"category" :    "Repository",
			"team" :        "Bologna",
			"dependency" :  "Java",
			"update" :      "2008",
			"status" :      "active",
			"contact" :     "Marco Caprini, Emanuele Giordano",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "ARQ",
			"description" : "Query the Registry"
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/keplerclotho\/",
			"url" :         "http:\/\/sourceforge.net\/projects\/keplerclotho\/",
			"category" :    "Workflow Management, Assembly",
			"team" :        "Berkeley",
			"dependency" :  "Clotho",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Kepler-Clotho integration",
			"description" : "Development of actors and automated assembly processes for the scientific workflow management system Kepler. Kepler is a multi-university design effort focusing on scientific workflows, and is used by a wide variety of projects in different fields. What the Berkely team did was to integrate Kepler into Clotho, as part of the Clotho-Eugene-Spectacles-Kepler software package"
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/eugene\/develop",
			"url" :         "http:\/\/eugenecad.org\/",
			"category" :    "Language Definition",
			"team" :        "Berkeley",
			"dependency" :  "Java",
			"update" :      "2010",
			"status" :      "active",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Eugene",
			"description" : "Biological programming language to constrain the composition of genetic parts as a specification of biological constructs. One can both specify the structure of the design (its basic elements) as well composites of basic elements. A system of rules for part composition is also provided."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:Peking_S\/modeling\/lgatedesign",
			"category" :    "Circuit Design, Simulation",
			"team" :        "Peking",
			"update" :      "2011",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "Logic Gate Design",
			"description" : "Logic Gate Design software"
		},
		{
			"source" :      "https:\/\/github.com\/igemsoftware\/NYC_Software_2011",
			"url" :         "https:\/\/github.com\/igemsoftware\/NYC_Software_2011 and http:\/\/2011.igem.org\/Team:NYC_Software\/Tools\/BLAST",
			"category" :    "Repository",
			"team" :        "NYC",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Russell Durrett",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Blast",
			"description" : "BLAST executables to find the most similar parts (or parts of parts) in the current registry"
		},
		{
			"source" :      "http:\/\/2010.igem.org\/wiki\/images\/c\/ca\/TUM2010_Software_src.zip",
			"url" :         "http:\/\/2010.igem.org\/Team:TU_Munich\/Software",
			"category" :    "DNA Design",
			"team" :        "TU Munich",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Friedrich Simmel",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "connect BioBricks",
			"description" : "Generate a single DNA plasmid that contains the custom network entered as input. The software uses a set of predefined DNA designs for AND, OR and NOT gates to generate the gene circuit. A selected few BioBricks serve as input and ouput to the network. In principle, any (of at least 200) transcription-regulating BioBrick can be used as an input, and any gene can serve as an output."
		},
		{
			"source" :      "https:\/\/github.com\/jcnossen\/InteractionHomologMapping",
			"url" :         "http:\/\/2010.igem.org\/Team:TU_Delft#page=Software\/im-tutorial",
			"category" :    "Protein Interaction Prediction",
			"team" :        "TU Delft",
			"dependency" :  "STRING database, Interaction Mapping, Cytoscape",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jelmer Cnossen",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Interaction Mapping Application",
			"description" : "Find putative interactions of proteins in a new host organism."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:BU_Wellesley_Software\/Trumpet",
			"category" :    "Circuit Design",
			"team" :        "BU-Wellesley",
			"dependency" :  "Windows, Microsoft Surface",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Trumpet",
			"description" : "A Trumpet testbed comprises a library of genes and promoters, which can be configured into any desired permutation or combination by treating the DNA with a sequence of enzymes called recombinases. Our design algorithm takes a set of DNA parts, and produces a testbed design and enzyme key dictionary, such that the testbed can be configured into any sequence of promoters and genes by treating it with the corresponding enzyme key. This allows us to rewire the genes and switches and study all their combinations. The goal of Trumpet is to aid the user in designing fully reconfigurable genetic circuit plasmids."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Bologna\/Software#2",
			"category" :    "Image Analysis",
			"team" :        "Bologna",
			"dependency" :  "MATLAB",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Emanuele Giordano, Giulio Zanaroli",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "VIFluoR - Very Inexpensive Fluorescence Reader)",
			"description" : "Estimate, by imaging analysis, the fluorescence emitted by a single bacterium."
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:Paris_Liliane_Bettencourt\/Project\/SIP",
			"category" :    "Various",
			"team" :        "Paris-Liliane Bettencourt",
			"dependency" :  "Sqlite 3",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Ariel Lindner",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "SIP Wiki Analyser",
			"description" : "Analysis of Statistically Improbable Phrases of iGEM wikis"
		},
		{
			"source" :      "https:\/\/github.com\/igemsoftware\/METU-BIN_Ankara_2011",
			"url" :         "http:\/\/2011.igem.org\/Team:METU-BIN_Ankara\/Project#Results",
			"category" :    "Integrated Design",
			"team" :        "METU BIN Ankara",
			"update" :      "2011",
			"status" :      "finished",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "Mining for BioBricks",
			"description" : "Design genetic constructs according to their input and output parameters. A web based tool"
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Slovenia\/Software (may contain malware)",
			"category" :    "Nanotechnology, Protein Structure",
			"team" :        "Slovenia",
			"update" :      "Not found",
			"status" :      "not found",
			"contact" :     "Roman Jerala",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "nanoBricks-pro",
			"description" : "Constructing structures similar to origami, made out of polypeptides. In their page, they claim to have designed software to predict the stability of these structures--coils--, but the link to the \"software\" section reads \"...to be added soon\"."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/9\/9f\/Simbiology2Latex.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:KULeuven\/Software\/Simbiology2LaTeX",
			"category" :    "Converter, Modelling",
			"team" :        "KU Leuven",
			"dependency" :  "MATLAB",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Bart De Moor,Georges Gielen, Kathleen Marchal, Yves Moreau, Bart Nicola\u00EF, Johan Robben, Jos Vanderleyden",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Simbiology2LaTeX",
			"description" : "Convert Matlab Simbiology models to LaTeX-code. This code can be easily converted to a pdf-file with various open source programs (e.g. TeXnicCenter for Windows). The idea for this toolbox is based on the plugin Squeezer for CellDesigner."
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:Mexico-UNAM-CINVESTAV\/Software",
			"category" :    "Various",
			"team" :        "Mexico UNAM-CINVESTAV",
			"dependency" :  "Not found",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Elias Samra-Hassan, Francisco Javier Razo Hern\u00E1ndez",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "BioBricks jTools",
			"description" : "Utilities to automate biobricks from files"
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:Potsdam_Bioware\/Software#BioLog_App",
			"category" :    "Lab notebook",
			"team" :        "Potsdam",
			"update" :      "2011",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "BioLog app",
			"description" : "an app where you can upload your protocols and which guides you during your work step by step?"
		},
		{
			"source" :      "https:\/\/github.com\/igemsoftware\/ENSPS-Strasbourg_2011",
			"url" :         "http:\/\/2011.igem.org\/Team:ENSPS-Strasbourg",
			"category" :    "Modelling",
			"team" :        "ENSPS-Strasbourg",
			"dependency" :  "Dolphin Smash",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Jacques Haeich",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "BrioBricks Models Generator for electronic Simulator (BBMGS)",
			"description" : "A graphical user interface for designers of synthetic biosystems. Helps in their design process by simulating the system. The midterm objective is to base the software on the database of biological material, fed by other iGEM teams. It will model the studied system in an electronic modeling language and provide simulation results to the user."
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:TU_Delft#page=Education\/game",
			"category" :    "Game, Educational",
			"team" :        "TU Delft",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "video",
			"contact" :     "Marin Licina, Martijn Rentmeester, Sverre Rabbelier, Thomas Rens, Michel de Ridder",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Pimp Your Bacterium",
			"description" : "iGEM Educational Game in which the player controls a bacterium that needs the appropriate properties and at the same time has to survive viruses, antibiotics and low oxygen levels. Creating these new properties is done by collecting parts in your action screen. If you have collected enough parts, you can go to the lab to your inventory and assemble the needed BioBricks. Children can learn that adding properties can be done by combining parts of DNA and get to know the factors that play a role in the survival of a microorganisms."
		},
		{
			"source" :      "https:\/\/github.com\/haydnKing\/Gibthon",
			"url" :         "http:\/\/gibthon.org\/",
			"category" :    "Design, Gibson Assembly",
			"team" :        "Cambridge",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Haydn King, Bill Collins",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gibthon Construct Designer2",
			"description" : "Facilitates the design of primers to create overlaps in your DNA for Gibson Assembly. It will optimise them for length and melting temperature, and warn you of any potential selfpriming or mispriming."
		},
		{
			"source" :      "https:\/\/github.com\/igemsoftware\/CBNU-KOREA_2011",
			"url" :         "http:\/\/2011.igem.org\/Team:CBNU-Korea\/Data\/Browser",
			"category" :    "Genome Design",
			"team" :        "CBNU-Korea",
			"dependency" :  "Java SDK, Eclipse",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Young-Chang Kim, Chung Sei Rhee, Sung Duck Lee",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Genome Organization Database and Designer (GOD)",
			"description" : "Design an artificial genome with essential genes in silico."
		},
		{
			"source" :      "http:\/\/2009.igem.org\/Team:Freiburg_software\/Code",
			"url" :         "http:\/\/2009.igem.org\/Team:Freiburg_software",
			"category" :    "Collaborative, Sequence Alignment, Sequence Analysis, Primer Design",
			"team" :        "Freiburg",
			"dependency" :  "Google Wave",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Raik Gruenberg",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "SynBioWave",
			"description" : "Based on Google's collaboration and communication tool Wave. Made for collaborative research comprising parts design and documentation. Synthetic biologists can record and share the process of creating research data and perform basic tasks using SynBioWave. This gives synthetic biology access to the collaborative and interactive web 2.0."
		},
		{
			"source" :      "https:\/\/github.com\/igemsoftware\/USTC-Software_2011",
			"url" :         "http:\/\/2011.igem.org\/Team:USTC-Software",
			"category" :    "Modelling",
			"team" :        "USTC",
			"dependency" :  "No",
			"update" :      "2011",
			"status" :      "not found",
			"contact" :     "Hao Jiang",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Lachesis",
			"description" : "A system to aid in the design, optimization, and simulation of parts circuits. The Assembly View graphical interface creates models using MoDeL, a rule-based modeling language.  The interface provides utilities to insert components (parts and species) into compartments or plasmids. Finally, the model can be \"solved\", an SBML file is created.."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:UEA-JIC_Norwich\/software and http:\/\/2011.igem.org\/File:Igem_defender.zip",
			"category" :    "Game",
			"team" :        "UEA-JIC Norwich",
			"update" :      "2011",
			"status" :      "active",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "iGEM Registry Defender",
			"description" : "a new Android game developed by UEA-JIC Norwich iGEM11 team. The aim of the game is to prevent poorly characterised bio bricks from entering the registry, whack-a-mole style"
		},
		{
			"source" :      "http:\/\/2011.igem.org\/wiki\/images\/f\/f5\/CodonOptimisationScript.zip",
			"url" :         "http:\/\/2011.igem.org\/Team:Imperial_College_London\/Software",
			"category" :    "Codon Optimization, Sequence Analysis",
			"team" :        "Imperial College London",
			"dependency" :  "R",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Guy-Bart Stan",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Joint codon optimisation algorithm",
			"description" : "Codon optimization: maintain codon diversity while simultaneously limiting rare codon inclusion. Achieved by weighting codon selection using bias tables obtained from the Codon Usage Database. Joint optimisation was facilitated by combining the bias tables of E. coli and B. subtilis. Following the generation of a seed-sequence, the stochastic pruning of rare codons was used to iteratively optimise the sequence."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/django.gibthon.org\/",
			"category" :    "Primer Design",
			"team" :        "Cambridge",
			"dependency" :  "PHP, web, Django, mfold, BioPython, jQuery",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jim Ajioka,Jim Haseloff, Gos Micklem",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Gibthon Beta",
			"description" : "Helps design primers for Gibson Assembly. Takes as an input the sequences surrounding the join for which you wish to design primers. You can interactively adjust the length of the primers to tune the melting temperature of the DNA and check for secondary structures."
		},
		{
			"source" :      "http:\/\/2007.igem.org\/Image:Igem_code.zip",
			"url" :         "http:\/\/2007.igem.org\/Calgary\/software",
			"category" :    "Printing",
			"team" :        "Calgary",
			"dependency" :  "PlotterPilot",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Christian Jacob",
			"free" :        "yes",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "PlotterPilot - routines",
			"description" : "To print on bacteria. Converts NS-Bezier paths into a series of objective C commands that could be translated into motion by a printer system. The system uses VHDL code to handle the FXS chip of our control ciruct."
		},
		{
			"source" :      "http:\/\/2007.igem.org\/Image:Chassic_Characterisation_Classic_Model.zip",
			"url" :         "http:\/\/2007.igem.org\/Imperial\/Dry_Lab\/Software",
			"category" :    "Data Analysis",
			"team" :        "Imperial",
			"dependency" :  "MATLAB",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Richard Kitney, Paul Freemont",
			"free" :        "yes",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "Chassis Characterisation with the Classic Promoter Model",
			"description" : "Load the results of several experiments at the same time. These data are then analysed by shape matching. Once the best fit is identified, the results are returned and the corresponding graphs are plotted."
		},
		{
			"source" :      "http:\/\/www.ehmn.bioinformatics.ed.ac.uk\/igem\/",
			"url" :         "http:\/\/2008.igem.org\/Team:Edinburgh\/Software",
			"category" :    "Circuit Design, Simulation",
			"team" :        "Edinburgh",
			"dependency" :  "Web",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Hongwu Ma, Alistair Elfick",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "MGEC - Modeling Genetically Engineered Circuits",
			"description" : "Automatically generates a preliminary model in SBML format for the gene circuit based on promoters and genes assembled in order to form the circuit. This model can then be analyzed in other software such as Copasi for further improvement. For the inexperienced user."
		},
		{
			"source" :      "https:\/\/github.com\/haydnKing\/Gibthon",
			"url" :         "http:\/\/gibthon.org\/",
			"category" :    "Design, Gibson Assembly",
			"team" :        "Cambridge",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Haydn King, Bill Collins",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gibthon",
			"description" : "A suite of web-based tools to aid in the design and manufacture of synthetic parts and devices for biological systems."
		},
		{
			"source" :      "http:\/\/igem.uwaterloo.ca\/biomortar-svn",
			"url" :         "http:\/\/igem.uwaterloo.ca\/BioMortar",
			"category" :    "Repository, Laboratory Protocols",
			"team" :        "Waterloo",
			"dependency" :  "ASP.NET, PostgreSQL",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Andre Masella",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "BioMortar - Freezer Management",
			"description" : "Freezer management of BioBricks. It understands standard assembly and can gather information from the Parts Registry. geared to iGEM and Standard Assembly so it keeps track of BioBrick part numbers and also knows that some parts are composite parts made of other parts. The advantage of this is that it can construct sequences for composite parts automatically. Although Standard Assembly makes it easy to ligate parts, it is fairly easy to reverse the enzymes accidentally. One of the goals of BioMortar was to generate prepared protocols with the enzymes filled in to prevent such errors. It also could do the math to calculate the quantities of solutions needed."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/p\/partsrobot\/svn\/36\/tree\/",
			"url" :         "http:\/\/sourceforge.net\/p\/partsrobot\/home\/Home\/",
			"category" :    "Part Submission, Part Documentation",
			"team" :        "University of Washington",
			"dependency" :  "no",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Herbert Sauro",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "PartsRobot",
			"description" : "Helps iGEM teams and other researchers submit spreadsheets of standard biological parts to the Parts Registry at MIT. The current version has been tested by the Washington 2010 team, but we'd love to hear your experience too! If it works we'll feel good about ourselves, and if not we'll try to fix it."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Parts_Repository",
			"category" :    "Repository",
			"team" :        "Newcastle University",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Anil Wipat",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Parts Repository",
			"description" : "Stores information from the literature and various databases about biological parts such as promoters, ribosomal binding sites and protein coding regions. As well as biological information, this includes fragments of CellML that facilitate incorporation of the part into a larger simulation."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2009.igem.org\/Team:Calgary\/Modelling\/MC\/Intro",
			"category" :    "Simulation",
			"team" :        "Calgary",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Christian Jacob, Anders Nygren",
			"free" :        "not available",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "MCMGem",
			"description" : "Observe the behavior of each individual cell as well as the emergent properties of the whole population. It also enables us to manipulate factors involved in the simulation to understand their effects in individual as well as colony behaviors. Based on the Membrane Computing technique"
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:UIUC-Illinois-Software",
			"category" :    "DNA Design",
			"team" :        "UIUC-Illinois",
			"dependency" :  "Web",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Nathan Price, Kaustubh Bhalerao",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "BioMORTAR",
			"description" : "Design a plasmid in-silico for the user-specified host bacteria as well as model the growth of the designed cell. Three Part Genetic Design Suite composed of IMPtools 2.0, Plasmid Designer, and COBRA Cell Modeling"
		},
		{
			"source" :      "http:\/\/code.google.com\/p\/arcanum-project\/",
			"url" :         "http:\/\/2011.igem.org\/Team:Sevilla\/Software\/GPSE",
			"category" :    "Simulation, Modelling",
			"team" :        "Sevilla",
			"update" :      "2011",
			"status" :      "under development",
			"contact" :     "Eduardo Villalobo, Francisco Romero, Ana Calvo",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "GPSE - Graphical P-System Editor",
			"description" : "Design biological P-Systems. Makes the preparation of the files needed for the P-Systems simulators easier, more intuitive."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:Rutgers\/MYS!S and https:\/\/github.com\/RutgersGEARS\/iGEM-Rutgers-Software",
			"category" :    "Repository",
			"team" :        "Rutgers",
			"update" :      "2011",
			"status" :      "under development",
			"type" :        "Item",
			"year" :        "2011",
			"label" :       "MYS!S",
			"description" : "MYS!S is a stand alone software tool designed to assist an \u201CiGEMer\u201D when designing a new BioBrick part or modifying an existing one."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/synbioss\/develop",
			"url" :         "http:\/\/synbioss.sourceforge.net\/",
			"category" :    "Modelling, Circuit Design",
			"team" :        "Minnesota",
			"dependency" :  "Web",
			"update" :      "2010",
			"status" :      "active",
			"contact" :     "Yiannis Kaznessis",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "SynBioSS Designer",
			"description" : "Transform a sequence of BioBricks, or any other set of biomolecular components, to a set of reactions that can be simulated dynamically in SynBioSS DS. The user simply inputs the Biobricks and their relationships and Designer builds a reaction network using biological interactions rules."
		},
		{
			"source" :      "http:\/\/phyrtech.com\/igem\/Labnote_dist.zip",
			"url" :         "http:\/\/2011.igem.org\/Team:Minnesota\/Software",
			"category" :    "Electronic Laboratory Notebook",
			"team" :        "Minnesota",
			"dependency" :  "Microsoft .NET Framework 4.0",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Nathan Davis",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Labnote",
			"description" : [
				"An electronic laboratory notebook (ELN) which is specifically tailored to interact with the iGEM Wiki page. Uses Rtf2WikiMarkup: a command-line tool designed to convert the RTF entries produced by Labnote into valid Wiki Markup suitable for upload to a MediaWiki-powered site",
				"and WikiBot: a command-line tool which allows automated uploading of files and pages to a MediaWiki-powered site. This application utilizes a modified version of the DotNetWikiBot"
			]
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:BU_Wellesley_Software\/Puppetshow",
			"category" :    "Laboratory Automation",
			"team" :        "BU-Wellesley",
			"dependency" :  "Windows, Microsoft Surface",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Puppetshow",
			"description" : [
				"Laboratory protocols often require careful and precise execution of numerous procedures to achieve desired results. However, slight inaccuracies and miscalculations attributable to human-error continually arise and often lead to invalid conclusions. For this reason, we have developed an accurate workflow that executes biological protocols in a more systematic manner. The solution we propose is the Puppeteer Biological Protocol Automation Suite. Our suite centers on the execution of Puppeteer, a high-level protocol specification language used to programmatically describe biological protocols",
				"it is executed on a liquid-handling robot with minimal user intervention. PuppetShow allows users to create protocols locally, but will eventually incorporate a \u201CProtocol Repository\u201D that will retain version-controlled instances of uploaded protocols. This feature will promote all forms of collaboration within the synthetic biology community, with a predominant focus on inter-laboratory protocol sharing."
			]
		},
		{
			"source" :      "https:\/\/code.google.com\/p\/setob\/",
			"url" :         "http:\/\/setob.blogspot.com\/",
			"category" :    "Repository",
			"team" :        "University of Washington",
			"dependency" :  "Flash",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Herbert Sauro",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "SeToB",
			"description" : "Search engine for a few parts and devices from the Standard Registry, uploaded to a PoBoL database."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/f\/ff\/MultiCell_Toolbox.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:KULeuven\/Software\/MultiCell",
			"category" :    "Modelling, Simulation",
			"team" :        "KU Leuven",
			"dependency" :  "MATLAB",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Bart De Moor,Georges Gielen, Kathleen Marchal, Yves Moreau, Bart Nicola\u00EF, Johan Robben, Jos Vanderleyden",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "MultiCell toolbox",
			"description" : "MATLAB Symbiology project in a multicellular environment. An easy-to-use graphical interface to configure the simulation settings and one click simulation."
		},
		{
			"source" :      "http:\/\/cambridgeigem.org\/genbank\/index.phps",
			"url" :         "http:\/\/2010.igem.org\/Team:Cambridge\/Tools\/GenBank",
			"category" :    "Repository",
			"team" :        "Cambridge",
			"dependency" :  "PHP, web, Django, mfold, BioPython, jQuery",
			"update" :      "Not found",
			"status" :      "finished",
			"contact" :     "Jim Ajioka,Jim Haseloff, Gos Micklem",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "BioBrick > GenBank",
			"description" : "Allows any BioBrick to be downloaded with annotations in GenBank format. This allows import into many DNA editing programs, e.g. ApE. You can now access the BioBrick > GenBank converter directly from the Parts Registry by going to Tools > GenBank format."
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:METU_Turkey_Software",
			"category" :    "Pathway Generation, Database Search",
			"team" :        "METU Turkey",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Ye\u015Fim Ayd\u0131n-Son, Tolga Can",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "BioGuide",
			"description" : "Searches for biobricks in the standard registry and produces a pathway based on a search query of input and output proteins."
		},
		{
			"source" :      "http:\/\/gcat.davidson.edu\/IGEM06\/oligo.txt",
			"url" :         "http:\/\/gcat.davidson.edu\/IGEM06\/oligo.html",
			"category" :    "DNA Design, RNA Design",
			"team" :        "Davidson-Missouri W",
			"dependency" :  "Web",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "A. Malcolm Campbell",
			"free" :        "yes",
			"year" :        "2006",
			"type" :        "Item",
			"label" :       "Lancelator",
			"description" : "Gene Synthesis Optimization Program"
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/synbioss\/",
			"url" :         "http:\/\/synbioss.sourceforge.net\/",
			"category" :    "Integrated Workflows, Circuit Design, Simulation",
			"team" :        "Minnesota",
			"dependency" :  "none",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Yiannis Kaznessis",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "SynBioSS - Synthetic Biology Software Suite",
			"description" : "Generate, store, retrieve and quantitatively simulate synthetic biological networks. Facilitates computational synthetic biology and consists of three independent components: the Desktop Simulator (DS), the Wiki, and the Designer."
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:BU_Wellesley_Software\/OptimusPrimer",
			"category" :    "Primer Design",
			"team" :        "BU-Wellesley",
			"dependency" :  "Windows, Microsoft Surface",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Optimus Primer",
			"description" : "Allows users to create a primer design from a selected gene. Users can then can BLAST designs and run tests based on alignment and Gibb's free energy. Both applications make use of the same algorithms and allow saving as BioBricks. It is designed for integration with Clotho and allows users to save designs as BioBricks. Users can select a number of bases and run tests to determine the sustainability of their designs."
		},
		{
			"source" :      "http:\/\/tinkercellextra.svn.sourceforge.net\/viewvc\/tinkercellextra\/",
			"url" :         "http:\/\/www.tinkercell.com\/plugins\/wikidust",
			"category" :    "Network Design, Part Documentation",
			"team" :        "University of Washington",
			"dependency" :  "TinkerCell",
			"update" :      "2012",
			"status" :      "active",
			"contact" :     "Herbert Sauro",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "WikiDust",
			"description" : "Uses the Standard Biological Parts knowledgebase (SBPkb) to download information about publicly available DNA components (parts) into TinkerCell. This makes it fast and easy to diagram genetic regulatory networks that include common promoters, coding sequences, etc. Second, WikiDust creates and publishes imagemaps based on TinkerCell diagrams. Parts for which a URL is available in TinkerCell are clickable in the final image. This streamlines research because detailed, up-to-date information about each part in the network is just a click away."
		},
		{
			"url" :         "http:\/\/2009.igem.org\/Team:Valencia\/Simulations",
			"category" :    "Cell Simulation",
			"team" :        "Valencia",
			"dependency" :  "MATLAB",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Pedro Fern\u00E1ndez de C\u00F3rdoba",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "Modelling Interface",
			"description" : "Simulation that predicts how intracellular calcium concentration changes through time in cardiomyocites and neurons, according to a set of given parameters."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/Team:iHKU\/software",
			"url" :         "http:\/\/2008.igem.org\/Team:iHKU\/software#imagej",
			"category" :    "Image Analysis",
			"team" :        "iHKU",
			"dependency" :  "ImageJ",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Chenli Liu, Xiongfei FU, Lei Shi, XiaoJing Ren",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "ImageJ - process",
			"description" : "Describes the steps needed to analyze an image in the ImageJ tool. For example, how to get intensity values from a gel image."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/gcat.davidson.edu\/iGEM07\/genesplitter.html",
			"category" :    "DNA Design, RNA Design",
			"team" :        "Davidson-Missouri W",
			"dependency" :  "Web",
			"update" :      "2007",
			"status" :      "finished",
			"contact" :     "Karmella Haynes",
			"free" :        "yes",
			"year" :        "2007",
			"type" :        "Item",
			"label" :       "Gene Splitting",
			"description" : "Finds 4 primers to split a gene, so that hixC sequence can be inserted within the target gene's coding sequence without altering protein function. hixC is the target for hin proteins to cleave DNA so that it can be re-annealed in the opposite configuration. This software tool was used to design a synthetic biology solution to the \"pancake flipping\" problem."
		},
		{
			"source" :      "http:\/\/sourceforge.net\/projects\/igemzjuchina\/files\/Bach_source.rar\/download",
			"url" :         "http:\/\/2010.igem.org\/Team:ZJU-China\/Software",
			"category" :    "Sequence Optimization",
			"team" :        "ZJU-China",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Zhejiang University",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Bach gene composer",
			"description" : "\"Bach is a gene composer software that optimizes coding sequences and calculates corresponding RiPS to illustrate their translation rates. In sequence optimization, Bach incorporates 4 distinct approaches from which users can freely choose: SYNONYMOUS SUBSTITUTION, SLOW, FAST and MATCH. In the last section of GET RIPS, users are provided with a quantitative view of the translational behavior of the input coding sequence and the four sequences produced by the four methods as well.\""
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2011.igem.org\/Team:Warsaw\/Software",
			"category" :    "Various",
			"team" :        "Warsaw",
			"dependency" :  "Eclipse, RBSCalculator, Nupack, ViennaRNA,",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Jacek Bielecki",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "ExpressionAdaptorsGenerator",
			"description" : "The Warsaw team developed software to generate and analyze \"Expression Adaptors\", a type of synthetic biology part that they created to deal with the variable expressibility and stochastic nature of RBS."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2011.igem.org\/Team:Dundee\/Software",
			"category" :    "DNA Sequence Design",
			"team" :        "Dundee",
			"dependency" :  "Android, iOS",
			"update" :      "2011",
			"status" :      "finished",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "The Lazy Scientist",
			"description" : "A translation tool aimed to assist synthetic biologists. The app has a number of functions including DNA to amino acid translation, back translation, reverse, complement and reverse complement. The app also contains a genetic codon lookup table allowing all corresponding codons for an amino acid to be looked up."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2010.igem.org\/Team:VT-ENSIMAG\/Genothreat",
			"category" :    "Sequence Analysis, Security",
			"team" :        "VT-ENSIMAG",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jean Peccoud",
			"free" :        "not available",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Genothreat",
			"description" : "Tests if the sequence is dangerous and needs further investigation or if the sequence can be synthesized. It follows the algorithm given by the federal guidelines: uses the Blast software for sequence alignment. Biojava is used to manage the sequence (six-frame translation, reversed frame, extracting sub-sequence), Written in Java. Works on both Windows and Linux distribution. Different versions optimize the time execution or the CPUs' utilization."
		},
		{
			"source" :      "https:\/\/docs.google.com\/open?id=0BwUav0RF6ukrZGFlZDg1NjItMGM5My00MDcxLTg4YTUtMjUzMjg0YzNkYzVk",
			"url" :         "http:\/\/www.sbolstandard.org\/specification\/extensions\/visual",
			"category" :    "Language Definition",
			"team" :        "Stanford",
			"update" :      "2012",
			"status" :      "active",
			"contact" :     "Cesar Rodriguez",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "SBOLv - Synthetic Biology Open Language Visual",
			"description" : "A graphical notation that supports biological device development. It provides a formal notation for describing the physical composition of basic parts into composite parts during the development of biological devices. It is targeted for use by biological engineers in forward engineering projects. It encourages and supports model-driven engineering. SBOLv is an extension of SBOL."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2008.igem.org\/Team:Newcastle_University\/Evolutionary_Algorithm",
			"category" :    "Simulation, Optimization, Evolutionary Algorithm",
			"team" :        "Newcastle University",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Anil Wipat",
			"free" :        "not available",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "Evolutionary Algorithm",
			"description" : "Takes parts from the Parts Repository, assembles them randomly, but using constraints from the Constraints Repository, and evaluates the \"fitness\" of the resulting circuit depending on a desired input:output mapping. Equivalent parts with different parameters are swapped in and out at random, and fitter circuits selected in an iterative manner."
		},
		{
			"source" :      "https:\/\/github.com\/haydnKing\/Gibthon",
			"url" :         "http:\/\/gibthon.org\/",
			"category" :    "Assembly Planning, Gibson Assembly",
			"team" :        "Cambridge",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Haydn King, Bill Collins",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gibthon Molarity Calculator",
			"description" : "Molarity Calculator for Gibson Assembly"
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/2011.igem.org\/Team:Dundee\/dnaapp, http:\/\/2011.igem.org\/Team:Dundee\/Software",
			"category" :    "Codon Optimization, DNA Sequence Design",
			"team" :        "Dundee",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "finished",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gene Synthesiser",
			"description" : "Enables codon optimization by codon usage pattern-matching. The application analyses the patterns of codon usage of the gene that one wants to express, in its native organism and generates new synthetic DNA sequence optimised for the codon usage of the new target organism, yet matching any patterns of rare to common codons. Gene Synthesiser utilises codon usage data for over 35000 organisms."
		},
		{
			"source" :      "http:\/\/subversion.assembla.com\/svn\/brickmanager\/",
			"url" :         "http:\/\/brickmanager.appspot.com\/",
			"category" :    "Parts Design, Protocol Generation",
			"team" :        "Warsaw",
			"dependency" :  "Web",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jacek Bielecki",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "BioBrick Manager",
			"description" : "\"BioBrick Manager is extremely useful tool which helps you with planning your experiments or preparing printable instructions for others. It allows the user to download biobricks from partsregistry.org, connect them in order to obtain new parts, save resulting bricks as XML files and - what is the most important thing - to get protocol for creating that particular biobrick!\""
		},
		{
			"url" :         "http:\/\/2011.igem.org\/Team:BU_Wellesley_Software\/G-nomeSurferPro",
			"category" :    "Sequence Analysis, Literature",
			"team" :        "BU-Wellesley",
			"dependency" :  "Microsoft Surface",
			"update" :      "2011",
			"status" :      "finished",
			"contact" :     "Douglas Densmore",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "G-Nome Surfer Pro",
			"description" : "An integrated environment for viewing prokaryotic genomic data and literature. Users can find genes on the chromosome wheel or search for a gene name or Genbank number, access Genbank notes and publications from Pubmed and view alignment and translation from the chromosome bar. The extended desktop provides distraction free workspace for information processing and hypothesis formation. The Microsoft Surface allows users to collaborate in pairs and teams on design and research tasks."
		},
		{
			"source" :      "http:\/\/www.cbs.dtu.dk\/services\/PHUSER\/download.php",
			"url" :         "http:\/\/www.cbs.dtu.dk\/services\/PHUSER\/",
			"category" :    "Primer Design",
			"team" :        "DTU Denmark",
			"dependency" :  "Web, Python, Perl",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Kiran Raosaheb Patil, Rasmus Wernersson",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "PHUSER - Primer Help for USER",
			"description" : "Design of PCR optimized primers for USER (Uracil Specific Excision Reagent) fusion of custom DNA fragments."
		},
		{
			"source" :      "http:\/\/2009.igem.org\/Team:Washington-Software\/Downloads",
			"url" :         "http:\/\/2009.igem.org\/Team:Washington-Software\/Downloads",
			"category" :    "Hardware Driver, Robotics",
			"team" :        "University of Washington",
			"dependency" :  "ROBOTC",
			"update" :      "2009",
			"status" :      "finished",
			"contact" :     "Herbert Sauro",
			"free" :        "yes",
			"year" :        "2009",
			"type" :        "Item",
			"label" :       "BioBrick-A-Bot",
			"description" : "A cheap alternative to Commercial Liquid Handling Systems. The robot was built using Lego bricks, and is designed to be modular in design, allowing for user-designed parts and added functionality. Source code for the drivers that operate the BioBrick-A-Bot."
		},
		{
			"source" :      "https:\/\/github.com\/haydnKing\/Gibthon",
			"url" :         "http:\/\/gibthon.org\/",
			"category" :    "Primer Design, Gibson Assembly",
			"team" :        "Cambridge",
			"dependency" :  "Web",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Haydn King, Bill Collins",
			"free" :        "yes",
			"year" :        "2011",
			"type" :        "Item",
			"label" :       "Gibthon Primer Calculator",
			"description" : "Easy to use web based tool for primer design for Gibson Assembly, reducing the chance of errors while designing primers. While some degree of common sense will always be required, Gibthon will reduce the chance of making mistakes while designing these primers."
		},
		{
			"url" :         "http:\/\/2010.igem.org\/Team:TU_Delft#page=Software\/part-search",
			"category" :    "Registry, Query, Webservice",
			"team" :        "TU Delft",
			"dependency" :  "No",
			"update" :      "2010",
			"status" :      "finished",
			"contact" :     "Jelmer Cnossen",
			"free" :        "yes",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Partsregistry Query Service",
			"description" : "A part information service enables browsing parts by category, as specified by each part in the partsregistry editor. It also supports a range of other webservices that can be used by javascript or other applications."
		},
		{
			"source" :      "http:\/\/2008.igem.org\/wiki\/images\/a\/a2\/RNAHairpinDesigner.zip",
			"url" :         "http:\/\/2008.igem.org\/Team:TUDelft\/Temperature_software",
			"category" :    "RNA design",
			"team" :        "TU Delft",
			"update" :      "2008",
			"status" :      "finished",
			"contact" :     "Alexander Gultyaev, Filippo Menolascina",
			"free" :        "yes",
			"year" :        "2008",
			"type" :        "Item",
			"label" :       "RNA Hairpin Designer",
			"description" : "Designs RNA sequences that will form a hairpin structure with a given stability profile. A temperature sensitive hairpin can be designed automatically."
		},
		{
			"source" :      "not available",
			"url" :         "http:\/\/django.gibthon.org\/",
			"category" :    "Primer Design",
			"team" :        "Cambridge",
			"dependency" :  "PHP, web, Django, mfold, BioPython, jQuery",
			"update" :      "2011",
			"status" :      "active",
			"contact" :     "Jim Ajioka,Jim Haseloff, Gos Micklem",
			"free" :        "not available",
			"year" :        "2010",
			"type" :        "Item",
			"label" :       "Gibthon Construct Designer",
			"description" : "Design your construct from its component genes and automatically generate the optimal primers. The tool has access to the Parts Registry Database, the NCBI's Nucleotide database, as well as accepting genbank formatted files from the user. The user is able to store constructs and their own genes for recovery later. A powerful Gibthon tool."
		}
	]
}
